web/src/Search.css

```diff
@@ -0,0 +1,55 @@
+html {
```

Comment:
@elkmare these can be moved to `index.css` I think

Reply:
Done :)